services/elasticsearch/elasticsearch.go

```diff
@@ -314,9 +317,10 @@ func (d *dedicatedElasticsearchAdapter) checkElasticsearchStatus(i *Elasticsearc
 			// Instance not up yet.
 			return base.InstanceNotCreated, errors.New("Instance not available yet. Please wait and try again..")
 		}
+	} else {
+		return base.InstanceNotCreated, nil
 	}
```

Comment:
```suggestion
	    return base.InstanceNotCreated, nil
```

The `else` is unnecessary, I believe. Everything inside the `if` returns early, so if we have reached the outside of the `if`, then we can just return the default